bedrock/pocket/templates/pocket/save-to-pocket.html

```diff
@@ -66,25 +68,25 @@ <h2 class="sub-section-heading">Absorb content in a quiet space</h2>
   media_class='mzp-l-split-h-center',
   image_url='img/pocket/focused-reading.png',
   )%}
-  <h2 class="sub-section-heading">Customize your experience</h2>
+  <h2 class="sub-section-heading">{{ ftl('pocket-save-customize-your') }}</h2>
   <ul class="section-lede">
-    <li>Tailor text sizes, font styles, or viewing mode for focused reading.</li>
-    <li>Categorize saves with tags, and mark key passages with highlights.</li>
-    <li>Listen to articles on the Pocket app with our audio playback option.</li>
+    <li>{{ ftl('pocket-save-tailor-text') }}</li>
+    <li>{{ ftl('pocket-save-categorize-saves') }}</li>
+    <li>{{ ftl('pocket-save-listen-to') }}</li>
   </ul>
-  <a href="https://getpocket.com/signup" class="green-line-cta">Get started with Pocket</a>
+  <a href="https://getpocket.com/signup" class="green-line-cta">{{ ftl('pocket-save-get-started-with') }}</a>
   {% endcall %}
 </section>
 
 <section class="personal-library">
   <h2 class="section-heading">
-    Start building your personal library now
+    {{ ftl('pocket-save-start-building') }}
   </h2>
   <div class="library-ctas">
     <div>
-      <a href="https://getpocket.com/signup" role="button" class="red-cta">Get started</a>
+      <a href="https://getpocket.com/signup" role="button" class="red-cta">{{ ftl('pocket-save-get-started') }}</a>
```

Comment:
Nit: remove `role="button"` from links. The `role` attribute implies behaviour expected, rather than an association with how an element is visibly styled?